README.md

```diff
@@ -308,4 +308,18 @@ applications, as well as several libraries and support applications.
    instance of ECHO. It will only provide the parts necessary to enable
    integration testing. You should not expect a perfect or complete
    implementation of ECHO.
-  - Main method: cmr.mock_echo.runner
\ No newline at end of file
+  - Main method: cmr.mock_echo.runner
+  
+# License
```

Comment:
should be an h2, only one h1 should be in the document. Also, this is the second License section